src/core/static-layout.js

```diff
@@ -31,7 +31,7 @@ export const naturalDimensions_ = {
   'AMP-PIXEL': {width: '0px', height: '0px'},
   'AMP-ANALYTICS': {width: '1px', height: '1px'},
   // TODO(dvoytenko): audio should have width:auto.
```

Comment:
@deepaklalwani97 Please remove the TODO.

@dvoytenko Since the TODO is assigned to you, do you have any thoughts on the PR?